### PR TITLE
[doc] Add links to Release announcements

### PR DESCRIPTION
**Pull request by ddhodge**
Add links to Release announcements

@netlify /preview/releases/

### CODE REVIEW

File: docs/content/preview/releases/yba-releases/v2.19.md [hunk 1]
```diff
@@ -14,11 +14,7 @@ rightNav:
 type: docs
 ---
 
-{{< tip title="YugabyteDB Anywhere release notes have moved" >}}
-The release notes for YugabyteDB Anywhere have moved here. The [YugabyteDB v2.19 release notes](../../ybdb-releases/v2.19/) (and all other releases) are still available in their original location.
-{{< /tip >}}
-
-What follows are the release notes for all releases in the **YugabyteDB Anywhere** v2.19 series. Content will be added as new notable features and changes are available in the patch releases of the YBA v2.19  series.
+What follows are the release notes for all releases in the **YugabyteDB Anywhere** v2.19 series. Content will be added as new notable features and changes are available in the patch releases of the YBA v2.19 series.
```
Comment (aishwarya24) [inline docs/content/preview/releases/yba-releases/v2.19.md:17]: Release announcements are missing for 2.19

**Review by aishwarya24 — COMMENTED** [review]
left some comments

File: docs/content/preview/releases/yba-releases/v2.19.md (hunk 1 above)
Comment (ddhodge) [reply docs/content/preview/releases/yba-releases/v2.19.md:17]: There was zero about YBA in the announcement...

**Comment by aishwarya24** [comment]
LGTM.

**Pull request merged**